cli/src/trade/trade.tsx

```diff
@@ -34,23 +46,41 @@ export enum TradeStage {
   Confirm = 7
 }
 
+const pool = getOption('pool')
```

Comment:
I think it is best to capitalize first letter of file-scope global variables, so they are clearly distinguished from component state variables